FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -117,6 +141,24 @@ get_options()
 	done
 
 	local error_count=0
+
+    if [ -z "${g_user}" ]; then
+        log_Err "user (--user=) required"
+        error_count=$(( error_count + 1 ))
+    else
+        log_Msg "g_user: ${g_user}"
+    fi
+
+    if [ -z "${g_password}" ]; then
+        log_Err "password (--password=) required"
+        error_count=$(( error_count + 1 ))
+    else
+        log_Msg "g_password: *******"
+    fi
+
+    if [ -z "${g_server}" ]; then
+        g_server="${default_server}"
```

Comment:
You are setting the value of `g_server` to the value of `default_server`, but I'm pretty sure that `default_server` has not been assigned a value. So I think that this would just set g_server to an empty string.

I suggest you change the `default_server` variable to be named g_default_server and define its value near the top of the script, right after `g_script_name` is defined. Give it whatever default value you want to use for the server in the case in which it is not specified on the command line.